### PR TITLE
Issue #3906: update documentation for usage of "period" property at S…

### PR DESCRIPTION
**Pull request by sagar-shah94**
Issue: #3906

### CODE REVIEW

File: src/xdocs/config_javadoc.xml [hunk 1]
```diff
@@ -1631,6 +1631,43 @@ public boolean isSomething()
     &lt;property name=&quot;period&quot; value=&quot;STRING_LITERAL&quot;/&gt;
```
Comment (MEZk) [inline src/xdocs/config_javadoc.xml]: @romani @rnveach 
This example is invalid.
See http://rveach.no-ip.org/checkstyle/checkstyle.php?action=view&config=abeb8b079dd5d1ed2d2c10f81fc65d12&code=0db8c02c469b9086140536de29d1f527&checkstyle=checkstyle-7.6-all.jar&printTree=

Should be
```xml
<module name="SummaryJavadocCheck">
    <property name="period" value="."/>
</module>
```

Looks like a copy-paste error.

@sagar-shah94 
Please, fix this example.

**Comment by sagar-shah94** [comment]
@MEZk done.

**Comment by romani** [comment]
I do not see a reason in example of default  value, please make example base on Chinise period.
See #2989 .

**Comment by sagar-shah94** [comment]
@romani in example i should use chinese period instead of period operator.Am i right?

**Comment by MEZk** [comment]
@sagar-shah94 
Yes, you are right.

**Comment by sagar-shah94** [comment]
@MEZk done

File: src/xdocs/config_javadoc.xml [hunk 2]
```diff
@@ -1628,9 +1628,46 @@ public boolean isSomething()
...
+public class TestClass {
+    void method() {
+    }
+}
+          ]]></source>
+            </td>
+            <td>
+              <source><![CDATA[
+/**
+ * This is invalid java doc
```
Comment (romani) [inline src/xdocs/config_javadoc.xml]: please put there simple "." to show clearly that default does not work if user defined smth custom.

**Comment by sagar-shah94** [comment]
@romani added "." in invalid java doc so default does not work.Please approve and merge.

**Comment by romani** [comment]
@sagar-shah94 , we do have all examples in non-table format, let continue that format.
please make one source example with two method(with valid and invalid javadoc)

**Comment by sagar-shah94** [comment]
@romani changes made.

**Comment by sagar-shah94** [comment]
@romani why continuous-integration/travis-ci/pr is failing it is not showing any logs.i have made all required changes please approve and merge.

**Comment by romani** [comment]
@rnveach , I am ok with changes, please do final review.

**Pull request merged**